### ISSUE TITLE
Can't drag items to own collection sometimes

### ISSUE THREAD

**Issue by corinnecurcie** [Japan-Digital-Archives/Japan-Digital-Archive#718]

**Comment by corinnecurcie** [comment]
Only works when you're not logged in?? But not always?? Certain angle??

**Comment by corinnecurcie** [comment]
https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/blob/51362f81088abb7f11f6bd0eec5c272aea659632/web/js/app/views/collections/jda.view.collection.my-collections-drawer.js

**Comment by jcuanm** [comment]
I could not find the file "jda.view.collection.my-collections-drawer.js" inside of the collections directory. The only file inside of the collections directory is "jda.collection.item.js". Was "jda.view.collection.my-collections-drawer.js" moved somewhere else?

**Comment by corinnecurcie** [comment]
Hey Javier, I just searched for the file name with github's search bar, and I saw in widget.js that the file is listed as being in the "../app/views/collections" folder. Because that's coming from a js file, the ".." refers to the js folder on our website (sometimes this is tricky). But I checked, and yes, the file is inside var/www/jda/web/js/app/views/collections. 

![screen shot 2015-01-05 at 4 29 23 pm](https://cloud.githubusercontent.com/assets/3768712/5620243/09603598-94f8-11e4-9e42-498f8114b72e.png)
